src/org/loklak/api/search/MeetupsCrawlerService.java

```diff
@@ -63,7 +63,7 @@ public JSONObject serviceImpl(Query call, HttpServletResponse response, Authoriz
 	}
 
 	public static SusiThought crawlMeetups(String url) {
-		
```

Comment:
There is a lot of spacing replacements with new line. May be this is out of bound of the PR